enterprise/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/MorselRuntimeAcceptanceTest.scala

```diff
@@ -275,7 +274,58 @@ class MorselRuntimeAcceptanceTest extends ExecutionEngineFunSuite {
     val resultSet = asScalaResult(result).toSet
     resultSet.map(map => map("n.name")) should equal(Set("Satia42"))
     result.getExecutionPlanDescription.getArguments.get("runtime") should equal("MORSEL")
+  }
```

Comment:
This whole class should be rewritten to use CypherComparisonSupport. We should not add more tests to this file, IMHO...